ci-operator/step-registry/optional-operators/README.md

```diff
@@ -0,0 +1,40 @@
+# Troubleshooting olm deployment failures
+
+This document should help you to analyze where to look in case that olm deployment has failed. It also describes workflows and which one should be your main concern. Logs location is also mentioned as one of the main subjects.
+* [Workflows](#Workflows):
+* [Logs and how to get them](#Logs-and-how-to-get-to-them):
+
+## Workflows:
+You can find all the workflows that are being executed in following yaml files:
+https://github.com/openshift/release/tree/master/ci-operator/config/redhat-openshift-ecosystem/cvp
+
+I will use `4.7` ocp version in this example:
+The workflow you are most interested in is : **optional-operators-cvp-common-aws**.
+
+*Env variables* together with *dependenciy_overrides* are specified before the workflow, these are used during the steps which are mentioned in workflow itself.
+`CVP-COMMON-AWS` workflow does following: installs an optional operator using the input index image, package and channel and executes a common suite of CVP tests to validate the optional operator.
+
+In case you want to know what individual step does, search for them in https://github.com/openshift/release/tree/master/ci-operator/step-registry/optional-operators
+More information about workflows, individual steps and explained content of folders can be found here: 
+https://gitlab.cee.redhat.com/cvp/wiki/-/blob/master/CVP%20Internal/DPTPProwJobs.md
+
+
+## Logs and how to get to them:
+If there is a problem with olm deployment in cvp product jenkins pipeline, specifically in *bundle-image-validation* job, such issue will be recorded to **operator-olm-deployment-bundle-image-output.txt** file which contains direct link to corresponding prow job.
+This link is gonna look like this: 
+https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-redhat-openshift-ecosystem-cvp-ocp-4.7-cvp-common-aws/"unique-number".
+
+In this prow job you can inspect what exactly went wrong, which part of the workflow failed, etc.
+
+I will describe investigation of the issue in few steps:
+
+**Step 1**: Log in product jenkins (bundle-image-validation) and look for optional-operator-olm-deployment-output.txt file in artifacts.
```

Comment:
I believe this document is geared towards end-users of the test, so they won't have direct access to the product jenkins. We probably want to start off from the test reports instead.

Reply:
Starting with prow tests report.